### ISSUE TITLE
Add skip to TOC/skip to content links

### ISSUE THREAD

**Issue by yatil** [w3c/tr-design#40]
As a visual keyboard user (on a large screen) the TOC is the first thing I see, yet the keyboard navigation doesn’t bring me to the TOC. Provide jump links to the TOC as the first thing in the site helps. That jump link should be invisible by default but visible on `:focus`.

(Relates to #39.)

**Comment by fantasai** [comment]
I've updated the style sheet to address to try to address these comments.
http://fantasai.inkedblade.net/style/design/w3c-restyle/2016/sample
Please play with it and file further issues as necessary.

...Note, however, there's one issue I can't easily fix: the blank link text of the unfocused link when the other one is focused by keyboard. Maybe I'll come up with a solution tomorrow. Or maybe it'll just look a bit awkward for now. :/
